x-pack/plugins/data_visualizer/public/application/common/components/field_data_row/action_menu/actions.ts

```diff
@@ -40,6 +52,37 @@ export function getActions(
         }
       },
       'data-test-subj': 'dataVisualizerActionViewInLensButton',
-    },
-  ];
+    });
+  }
+
+  // Allow to edit index pattern field
```

Comment:
I see in Discover you get the option to delete runtime fields that have been added to the index pattern? Did you look into providing that functionality? Not a requirement for this PR, but just wondered how much work that would involve?

Reply:
Note as far I'm aware, the delete action should only be added for runtime fields defined in the index pattern.

Reply:
Added the [delete index pattern field option](https://github.com/elastic/kibana/pull/101316/commits/1afeb8a23dfc135738efb4cb5d43f56a650923fc) here if it's a runtime field and if user has edit index pattern permission. Note that the current EUI behavior automatically makes it a dropdown menu if there's > 2 actions. [This seems to be a strict EUI guideline](https://elastic.github.io/eui/#/tabular-content/tables#adding-actions-to-table) so I think we should revisit where would be the best place for the `delete` button
<img width="1762" alt="Screen Shot 2021-06-14 at 20 24 07" src="https://user-images.githubusercontent.com/43350163/121979233-6bd28000-cd4f-11eb-883d-4ecf26c9ad9e.png">

Reply:
I think this looks ok, and follows the EUI guidelines with the ellipses icon used when there are more than 2 actions.